docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,148 @@
+ - **Switch to Bug Report Comparison**: This component allows you to compare and analyze two different bug report files.
+
+> [!NOTE]
+> The Battery Historian tool only supports **.txt** or **.zip** format for bug report file.
+
+## Use Battery Historian Tool
+
+The Battery Historian tool essentially requires a bug report file to process the battery consumption stats of a Tizen device. This bug report file contains all the relevant data regarding the battery consumption. You can generate the bug report using the SDB that dumps the battery data to your development machine and creates a report.
+
+The Battery Historian tool takes the bug report file as an input to analyze and compare the stats. After processing the report data, the Battery Historian uses HTML format to visualize the stats in a web browser.
+
+To use the Battery Historian tool, ensure that you have the bug report files readily available. If you do not have the bug report file with, you can generate one using the following procedure:
+
+### Generate Bug Report File
+
+To generate a bug report file, follow these steps:
+
+1. From your Linux Terminal, enter the following commands:
+
+    ```
+
+    $] sdb root on
+
+    $] sdb shell
+
+    ```
+
+2. From your Tizen device terminal, enter the following commands:
+
+    ```
+
+    sh-3.2] dumpsys batterstats \&gt; /tmp/battery\_dump.txt
+
+    sh-3.2] exit
+
+    ```
+
+3. From Linux Terminal, enter the following commands::
+
+    ```
+
+    $] sdb pull /tmp/battery\_dump.txt
+
+    ```
+
+After you execute the commands, the `batterystats.txt` file is generated in the specified directory using the optional path argument.
+
+> [!NOTE]
+> If no path is specified, the file gets created in your home directory.
+
+### View and Analyze Battery Data
+
+To view and analyze your data, follow these steps:
+
+1. In Battery Historian Main Window, click **Browse**.
+
+2. Select the appropriate bug report file.
+
+3. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in the file. After the analysis is done, a battery stats chart appears:
+
+**Figure: Battery Historian Chart**
+
+![](./media/graph1.png)
+
+The Battery Historian chart graph illustrates the battery consumption events over time, where usage of the various device components correlates to battery consumption.
+
+#### Interpret the Graph
+
+ - The vertical axis represents the battery percentage, while the horizontal axis represents time.
+ - The sloping black bar indicates your battery percentage level throughout the day.
+ - The colored line segments represent the various apps, services, and other elements that have used battery life.
+   - For instance, a green bar denotes an active time for WiFi.
+ - You can add more metrics, by clicking on Add Metric and the corresponding metric will appear on the chart. Some of the available metrics are  Charging status, Voltage, Device active, Plug and so on.
```

Comment:
Would you please check https://github.com/Samsung/tizen-docs/pull/1156#discussion_r449353888?

```suggestion
 - You can add more metrics, by clicking on **Add Metric** and the corresponding metric will appear on the chart. Some of the available metrics are  Charging status, Voltage, Device active, Plug and so on.
```